java/java.source.base/src/org/netbeans/api/java/source/TreeUtilities.java

```diff
@@ -124,7 +125,14 @@
      * @since 0.67
      */
     public static final Set<Kind> CLASS_TREE_KINDS = EnumSet.of(Kind.ANNOTATION_TYPE, Kind.CLASS, Kind.ENUM, Kind.INTERFACE);
-    
+    static {
```

Comment:
We could make CLASS_TREE_KINDS immutable, although not necessarily in this PR.

Reply:
Yes, we could. Will make immutable later.